src/main/resources/templates/testcase.html

```diff
@@ -1,28 +1,42 @@
 {#with it}
 <tr id="{subject}" about="{subject}" typeof="{typesList}">
-    <td><a href="{subject}">{subject}</a></td>
+    <td>
+        <a href="{subject}">{subject}</a>
+        <span rel="spec:requirementReference" resource="{requirementReference}"></span>
+    </td>
     <td property="dcterms:title">{title}</td>
-    <td property="spec:testScript">
+    <td>
         {#if isImplemented}
-        <a href="{testScript}">{testScript}</a>
+        <a rel="spec:testScript" href="{testScript}">{testScript}</a>
         {#else}Not yet implemented{/if}
     </td>
-    <td property="td:reviewStatus">{status}</td>
+    <td rel="td:reviewStatus" resource="{status}">
+        {#switch status}
+        {#case 'http://www.w3.org/2006/03/test-description#unreviewed'}Unreviewed
+        {#case 'http://www.w3.org/2006/03/test-description#accepted'}Accepted
+        {/switch}
+    </td>
     {#if coverageMode}
     <td>{isImplemented ? '✔' : '✗'}</td>
     {#else if isImplemented && assertion}
-    <td property="earl:subject"><a href="{assertion.testSubject}">{assertion.testSubject}</a></td>
-    <td property="earl:result">
+    <td rev="earl:test" resource="{assertion.subject}">
```

Comment:
This is going to be pretty nitpicky... and it is no big deal if you don't but might want to default to `rel` to express the statements, instead of `rev` :)